packages/solid-query/jest-preset.js

```diff
@@ -0,0 +1,7 @@
+const solidPreset = require('solid-jest/preset/browser/jest-preset')
+const tanStackPreset = require('../../jest-preset')
+
+module.exports = {
+  ...tanStackPreset,
+  ...solidPreset,
+}
```

Comment:
@ardeora @lukesmurray I'm having problems getting the solid tests to work in this PR. They are the only ones that use their own preset. I've merged the solid preset with our own preset, and I think that works.

I've had general troubles with `babel-jest` - it just gave me lots of "unexpected token" errors, as if it didn't transpile jsx / typescript at all. So I switched to ts-jest and things just work. But the solid preset also uses `babel-jest` under the hood as far as I can see, so the errors are still there for the solid package :/

Would appreciate if you could take a look here 🙏

Reply:
I don't really know what I'm doing, but adding a custom transformer (copied from the preset one + added more babel presets) made it work:

21fa9392

Reply:
Hello! Apologies for the late reply. I have been travelling internationally this weekend. So I will be AFK till Monday 😅 From a quick look. It seems like it breaks whenever it encounters type definitions/ types in the test file. Maybe @lukesmurray can take a look? If not I'll take a look first thing Monday

Reply:
let me know if you still need help with this. I'm not super familiar with solid's babel + jest setup but I can do my best to help if you're still running into issues.

Reply:
it works fine now, so this is resolved for me 🤷